self_hosting_machinery/webgui/static/style.css

```diff
@@ -763,4 +763,39 @@ h3 {
     100%{
         transform: translateX(100%);
     }
-}
\ No newline at end of file
+}
```

Comment:
Is there any way to inherit this file? without modifications

Reply:
.css file must be attached in a head. All what all tabs do is insert their HTML into index.html. Technically you can create a dedicated file and attach it into index.html, but we haven't done it before